src/main/java/wooteco/subway/ui/PathController.java

```diff
@@ -0,0 +1,24 @@
+package wooteco.subway.ui;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.dto.PathResponse;
+import wooteco.subway.service.PathService;
+
+@RestController
+public class PathController {
+
+    private final PathService pathService;
+
+    public PathController(final PathService pathService) {
+        this.pathService = pathService;
+    }
+
+    @GetMapping("/paths")
+    public ResponseEntity<PathResponse> showPaths(@RequestParam Long source, @RequestParam Long target,
+                                                  @RequestParam int age) {
+        return ResponseEntity.ok().body(pathService.createPath(source, target, age));
```

Comment:
```suggestion
        return ResponseEntity.ok(pathService.createPath(source, target, age));
```

200 응답을 보내줄 땐 이런 shortcut을 이용할 수 있어요. 또는, `ResponseEntity`로 감싸지 않고 응답 객체를 바로 리턴해도 됩니다.